global-api/migrations/versions/dc1837707630_ghgrp_table.py

```diff
@@ -0,0 +1,54 @@
+"""GHGRP_table
+
+Revision ID: dc1837707630
+Revises: 770cdfcf7a28
+Create Date: 2023-11-17 08:32:03.210927
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import text
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'dc1837707630'
+down_revision: Union[str, None] = '770cdfcf7a28'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "GHGRP_EPA",
+        sa.Column("id", sa.dialects.postgresql.UUID(as_uuid=True), primary_key=True),
+        sa.Column("Facility Id", sa.String, nullable=False),
+        sa.Column("Facility Name", sa.String, nullable=False),
+        sa.Column("City", sa.String, nullable=False),
```

Comment:
Inconsistent cases for column names. I would prefer all column names to be lowercase with underscores as separators (snake_case).